schema/maintenance/maintenance.go

```diff
@@ -33,6 +33,15 @@ type Maintenance struct {
 	LastHTTPError int      `json:"err,omitempty"`
 }
 
+type MaintenanceV1 struct {
+	ResourceTypes []string `json:"rt,omitempty"`
+	Interfaces    []string `json:"if,omitempty"`
+	Name          string   `json:"n,omitempty"`
+	FactoryReset  *bool    `json:"fr,omitempty"`
+	Reboot        *bool    `json:"rb,omitempty"`
+	LastHTTPError *int     `json:"err,omitempty"`
```

Comment:
Is this necessary? I understand that `false` is considered empty for a `bool` value with the omitempty tag, so the property won't be serialized. But is that a problem? Both `FactoryReset` and `Reboot` only trigger action when they are set to `true`, so if they are false the action won't be triggered with `Maintenance`. Is there some requirement for the properties to be present even if they are set to `false`?

Same with `LastHTTPError` and `0`.

Reply:
It will be nice to have properties of maintenance there as it is in iotivity-lite.
User know what need to be set. Otherwise you need to go the OCF spec or WoT resource.